components/admin_console/password_settings.jsx

```diff
@@ -258,26 +258,30 @@ export default class PasswordSettings extends AdminSettings {
                         </div>
                     </Setting>
                 </div>
-                <TextSetting
-                    id='maximumLoginAttempts'
-                    label={
-                        <FormattedMessage
-                            id='admin.service.attemptTitle'
-                            defaultMessage='Maximum Login Attempts:'
-                        />
-                    }
-                    placeholder={Utils.localizeMessage('admin.service.attemptExample', 'E.g.: "10"')}
-                    helpText={
-                        <FormattedMessage
-                            id='admin.service.attemptDescription'
-                            defaultMessage='Login attempts allowed before user is locked out and required to reset password via email.'
-                        />
-                    }
-                    value={this.state.maximumLoginAttempts}
-                    onChange={this.handleChange}
-                    setByEnv={this.isSetByEnv('ServiceSettings.MaximumLoginAttempts')}
-                    disabled={this.props.isDisabled}
-                />
+                {!this.props.config.ExperimentalSettings.RestrictSystemAdmin &&
```

Comment:
Is `ExperimentalSettings` always guaranteed in config?
Just to avoid any issue maybe:
`{!this.props.config.ExperimentalSettings?.RestrictSystemAdmin &&`